docs/projects/automate-project-git-source.ipynb

```diff
@@ -32,7 +32,7 @@
     "4. [Setting a workflow](#setting-a-workflow)\n",
```

Comment:
<p>need to change this ttps://github.com/your repo&nbsp;to https://github.com/mlrun/ci-cd-tutorial.git</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
fixed

Reply:
Approved